docs/openpool-proto.md

```diff
@@ -0,0 +1,71 @@
+# open pool v1
+
+## Fully decentralized token conversion protocol
+
+Given a `converter` contract, the following *must* be implemented.
+
+### Standard tables for `pools`
+
+```C++
+name symbols_get_index(symbol_code A, symbol_code B) {
```

Comment:
It can fail when the symbol names are too large:
eg: symbols_get_index("TLOSDAC", "HEART") --> Fails

Reply:
To fix this changed the return type of the function from `eosio::name` to `uint128_t`